### DISCUSSION TITLE
Promoted attributes Formatting

### DISCUSSION THREAD

**Discussion by marco10x15** [TriliumNext/Trilium#3655] [Q&A] [answered]
Hello, I've discovered this beautiful software and i'm in the process of learning to use it.
I have some question, that i put in different post in order to not messy the answers.

Is possible formatting the “Promoted attributes” in a table or in a personal view?
The Book's example has different attributes, it would be better if I can see:

on the first line the **author**

on the second line the attributes: **readingStart** - **readingEnd** - **link**

and in another line my personal attributes: **language** - **genre** - **series** - **series_num**

Thanks!

**Comment by zadam** [comment, marked as answer]
Hello, currently there's no way to change the formatting. You can only change the order.

**Reply by marco10x15** [reply]
Thanks.
